deltacat/tests/compute/compact_partition_rebase_then_incremental_test_cases.py

```diff
@@ -676,6 +687,7 @@ class RebaseThenIncrementalCompactionTestCaseParams(BaseCompactorTestCase):
         read_kwargs_provider=None,
         drop_duplicates=True,
         skip_enabled_compact_partition_drivers=None,
+        assert_compaction_audit=None,
```

Comment:
Why is this one None? Do we want to have a separate assert?

Reply:
My mistake, this one is no longer None.